README.md

```diff
@@ -186,6 +124,69 @@ print("Updated params:", m.get_params())
 Module.get_compiled_artifact(m).save("/tmp/some_file.vmfb")
 ```
 
+## What is a JAX program?
+
+The challenge with talking about extracting a JAX program is that in all
+generality, a JAX program is bounded only by what can run in the host Python
+interpreter. What we are looking for is a simpler definition that allows a
+useful set of standalone programs to be constructed and that meshes well with
+the programming model employed by typical JAX developers.
+
+The components that are the most interesting towards this end are:
+
+* `jax.jit` functions: stateless functions mapping inputs to outputs, natively
+  represented as JAXPR IR and universally convertible to MHLO IR.
```

Comment:
I'd like to unpack this as a newbie:

1. Is JAXPR an intermediate representation in MLIR or its own IR unbound to MLIR?
2. Is the path of lowering always going to be JAXPR IR -> MHLO IR or are/will there be other IRs?
3. What does "universally convertible" mean?

Reply:
* JAXPR is a pure Python, Jax specific representation for its programs. It doesn't leave Jax.
* The mechanism for lowering is actually by "interpreting" a Jax program. See jax.interpreters.mlir for the variant that builds an MLIR module (of MHLO).
* For Jax, it will always be to MHLO (possibly with other things added since MLIR makes this easy to mix and match).
* I was just trying to convey that MHLO is the native format of Jax, not a translation layer and they aim for full representability in MHLO (it isn't lossy).